packages/ricos-content-query/web/src/editor-query/queries/is-single-root-textblock.ts

```diff
@@ -0,0 +1,10 @@
+import type { QueryItem } from 'ricos-types';
+
+const NAME = 'isSingleRootTextBlock';
+const isSingleRootTextBlock: QueryItem =
+  ({ editorQuery }) =>
+  () => {
+    return !!editorQuery.selection().doc?.children?.[0].isTextBlock();
```

Comment:
This does not have to be single like this.. you should add
```suggestion
    return editorQuery.isMultipleSelection() && !!editorQuery.selection().doc?.children?.[0].isTextBlock();
```